packages/obojobo-document-engine/ObojoboDraft/Chunks/IFrame/render-settings.js

```diff
@@ -107,11 +105,16 @@ const getRenderSettings = (
 ) => {
 	const ms = model.modelState
 	const zoomValues = getZoomValues(mediaState, model)
-	const zoom = zoomValues.currentZoom
 	const setDimensions = getSetDimensions(ms, defaultWidth, defaultHeight)
 	const scaleAmount = getScaleAmount(actualWidth, padding, setDimensions.w)
 	const displayedTitle = getDisplayedTitle(ms)
-	const scaleDimensions = getScaleDimensions(ms, zoom, scaleAmount, minScale, setDimensions)
+	const scaleDimensions = getScaleDimensions(
+		ms,
+		zoomValues.currentZoom,
+		scaleAmount,
+		minScale,
```

Comment:
We might want to add a maximum scale as well since it is possible to zoom the iframe into an unusable state. (~ 150 clicks of the zoom button, or 15 sec of accidentally leaning on the enter button while zoom in is highlighted)
(This could possibly be fixed in a future issue)

Reply:
Added